src/components/Main/Results/AgeBarChart.tsx

```diff
@@ -85,19 +81,22 @@ export function AgeBarChart({ showHumanized, data, rates }: SimProps) {
                 }}
               >
                 <XAxis dataKey="name" />
-                <YAxis
-                  label={{ value: t('Cases'), angle: -90, position: 'insideLeft' }}
-                  tickFormatter={tickFormatter}
```

Comment:
Is this a merge issue?  The YAxis ticks should still be humanized to make them more readable/shorter

Reply:
There seems to have been some weirdness related to merging. I'm looking into it.